### DISCUSSION TITLE
I suggest vote system

### DISCUSSION THREAD

**Discussion by seunggabi** [community/community#13311] [Discussions]
# example

I will make table opposite `subscriber`

### vote
👍  decliner
👎  canceler
👀  subscriber_x
🚀  unsubscriber
😄  terminator

---

### vote
:keycap_one  decliner
:keycap_two   canceler
:keycap_three  subscriber_x
:keycap_four  unsubscriber
:keycap_five  terminator

### emoji button active 
- existsing emoji in this comment -> bottom of comment
  - keycap_one ~ five
<img width="306" alt="스크린샷 2022-03-23 오후 4 37 58" src="https://user-images.githubusercontent.com/8468371/159647459-fb1e9222-2f29-4de0-bfa7-c1e9fba9d210.png">

**Comment by k-young-passionate** [comment]
What a good idea!
